css/speakers-style.css

```diff
@@ -0,0 +1,213 @@
+.featuredSpeakersContainer {
+  background-color: #fff;
+  padding-top: 3rem;
+  padding-bottom: 2rem;
+}
+
+.featuredSpeakersContainer h2 {
+  color: #272a31;
+  text-align: center;
+  padding-bottom: 0.8rem;
+  font-size: 1.5rem;
+}
+
+.featuredSpeakersContainer hr {
+  margin: 0 42%;
+  border: 0.5px solid #ec5242;
+}
+
+.speakers {
+  padding-top: 2rem;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  padding-bottom: 1.5rem;
+}
+
+.speaker {
+  display: flex;
+  flex-direction: column;
+  padding-top: 0.5rem;
+  margin: 0 0.5rem;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  transition: 0.2s;
+}
+
+.speaker:hover {
+  transform: scale(1.01);
+  z-index: 3;
+}
+
+.imgCtn {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.speakerImg {
+  width: 8rem;
+  border-radius: 50%;
+}
+
+.speaker-detail {
+  padding: 0.5rem 1rem;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.speakerName {
+  font-size: 1rem;
+  font-weight: 600;
+  padding-bottom: 0.5rem;
+  text-align: center;
+}
+
+.speakerTitle {
+  color: #ec5242;
+  font-style: italic;
+  font-weight: 600;
+  text-align: center;
+  padding-bottom: 0.5rem;
+}
+
+.speakerDescription {
+  padding-top: 0.25rem;
+  font-size: 1rem;
+  text-align: center;
+}
+
+.speaker3,
+.speaker4,
+.speaker5,
+.speaker6 {
+  display: none;
+}
+
+.seeMoreBtn {
+  margin: 0 1rem;
+  padding: 0.5rem;
+  width: 95%;
+}
+
+.less {
+  margin: 0 1rem;
+  padding: 0.5rem;
+  width: 95%;
+}
```

Comment:
- [ ] Please make sure all elements are matching the template's design (Show More and Less Button).
Template design: ![image](https://user-images.githubusercontent.com/99830261/234926929-7927a5d8-c471-4f68-8e6f-e97d7b199441.png)
Your design: ![image](https://user-images.githubusercontent.com/99830261/234927171-5d55ea36-591d-4f3f-b640-040a0d6c5263.png)